app/settings/page.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { themeChange } from "theme-change";
+
+import PageTitle from "../../components/ui/PageTitle";
+
+const Settings = () => {
+  const themeValues = [
+    "light",
+    "cupcake",
+    "bumblebee",
+    "emerald",
+    "corporate",
+    "synthwave",
+    "retro",
+    "cyberpunk",
+    "valentine",
+    "halloween",
+    "garden",
+    "forest",
+    "aqua",
+    "lofi",
+    "pastel",
+    "fantasy",
+    "wireframe",
+    "black",
+    "luxury",
+    "dracula",
+    "cmyk",
+    "autumn",
+    "business",
+    "acid",
+    "lemonade",
+    "night",
+    "coffee",
+    "winter",
+  ];
+
+  const [theme, setTheme] = useState<string>();
+
+  useEffect(() => {
+    themeChange(false);
+  }, []);
```

Comment:
What does this do?

Reply:
I don't think it needs to be here and in providers, only providers (or maybe not at all?).

Reply:
(I hate useEffect so I always look for ways to get rid of it)

Reply:
The useEffect in provider uses the saved theme on any page when reloaded and the one in the settings page makes the selected option in the select list correct after choosing a theme. If necessary I can keep trying to find a way to get rid of them.

Reply:
we can refactor that later (but there is a way, I'll show you)